### ISSUE TITLE
Add support for 2000 and 1990 SF1s

### ISSUE THREAD

**Issue by josiekre** [transportfoundry/censusr#16]
Currently we only allow 2010 SF1 calls, which is not intuitive because we allow all years for ACS calls. Let's fix this.

**Comment by gregmacfarlane** [comment]
Do they even have that data in the API?

**Comment by josiekre** [comment]
Yes they do when I looked earlier today. Send like an easy feature to add.
But maybe not worth the time. Idk

**Comment by gregmacfarlane** [comment]
Well, if it's there and easy to add, then let's do it.
